### PR TITLE
Valid move queen

### PR DESCRIPTION
**Pull request by yanhackcode15**
- Added valid_move? in model/queen.rb
- Added rspec tests

### CODE REVIEW

**Comment by mscoutermarsh** [comment]
hey @yanhackcode15, it looks like there is code in here from the move pieces pull request.

Can you try to rebase, or start a new branch that only has the queen valid move code?

**Comment by yanhackcode15** [comment]
@mscoutermarsh see the new push. It has only the new cold.

File: app/models/queen.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class Queen < Piece
+	def valid_move?(destination_x, destination_y)
```
Comment (mscoutermarsh) [inline app/models/queen.rb]: you can delete this file

**Comment by yanhackcode15** [comment]
@mscoutermarsh I did what you suggested but still getting the no method find error for the test. Any tips?

**Pull request merged**